cmd/insights/verify-rule.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/fairwindsops/insights-cli/pkg/rules"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+var inputFile, outputFile string
+
+func init() {
+	verifyRuleCmd.PersistentFlags().StringVarP(&inputFile, "input", "i", "./input.yaml", "Input file")
+	verifyRuleCmd.PersistentFlags().StringVarP(&outputFile, "output", "o", "./output.yaml", "Output file")
+	rootCmd.AddCommand(verifyRuleCmd)
+}
+
+var verifyRuleCmd = &cobra.Command{
+	Use:   "verify-rule",
+	Short: "Verifies a rule against Insights",
+	Long:  "Verifies a rule to provided the action item and makes sure it will work fine testing it against Insights",
+	Run: func(cmd *cobra.Command, args []string) {
+		var verifyRule rules.VerifyRule
+		org := configurationObject.Options.Organization
+		host := configurationObject.Options.Hostname
+		in, err := os.Open(inputFile)
+		if err != nil {
+			exitWithError(fmt.Sprintf("Error when trying to read file %s", inputFile), err)
```

Comment:
`open file` would be a more precise error message at this point

Reply:
done